### ISSUE TITLE
Remove any non-essential analytics/bigquery access in sourcegraph-dev

### ISSUE THREAD

**Issue by davejrt** [sourcegraph/sourcegraph#18795]
The GCP-analytics account does not appear to be required in the sourecgraph-dev project, as all datasets and queries are executed in the telligentsourcegraph project. 

This access should be removed it if isn't required. 

The following steps will be taken based on a meeting with @pecigonzalo and @ebrodymoore 

- [x] reach out to Joe to determine how data is ingested into the telligentsourcegraph project from dot-com
- [x]  ensure the account in sourcegraph-dev is not used for any automated task that may break
- [ ] delete the analytics account

**Comment by davejrt** [comment]
Update from Joe:

Events are sent from dotcom via a pubsub -> dataflow -> BQ dataset. 

He isn't aware of any interaction between the gcp-analytics account in sourcegraph-dev and the datasets in the other project.

**Comment by pecigonzalo** [comment]
Nice, something for a separate issue, but we should terraform that workflow, particularly the pubsub component that probably lives in `sourcegraph-dev`

**Comment by davejrt** [comment]
I've removed all of the roles for this account, and backed them up. I'll leave the account like this for 24 hours to see if anything alerts or other users report something is broken

**Comment by dan-mckean** [comment]
@davejrt is this ready to finish off quickly?

**Comment by davejrt** [comment]
related note for later: https://sourcegraph.slack.com/archives/CJX299FGE/p1618412879030400

**Comment by davejrt** [comment]
I've backed this up and deleted it. Waiting for a shake down until monday then I'll close this issue

**Comment by davejrt** [comment]
After some discussion, this needs more work before we can close this. There should ideally be a read-only replica with the tables required from prod where we can execute queries, rather than prod directly. This can't be closed until that is completed.

**Comment by davejrt** [comment]
This account was never managed via terraform, so there is no PR as such here. The solution for this has been to create scheduled queries in the telligent project and copy the required tables into a dataset that bizops can access. 

The tables can be found here https://github.com/sourcegraph/sourcegraph/issues/19799#issuecomment-880840905
